### PR TITLE
feat: developing UI test for radio button.

### PR DESCRIPTION
**Pull request by Darsh0307**
GitHub Issue (If applicable): closes # 

## PR Type

What kind of change does this PR introduce?

- Bugfix
- Feature
- Code style update (formatting)
- Refactoring (no functional changes, no api changes)
- Build or CI related changes
- Documentation content changes
- Project automation
- Other... Please describe:

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested on iOS.
- [ ] Tested on Wasm.
- [ ] Tested on Android.
- [ ] Tested on UWP.
- [ ] Tested in both **Light** and **Dark** themes.
- [ ] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: Uno.Gallery/Uno.Gallery.UITest/Given_RadioButton_02_Fluent.cs [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+	{
+		/* This one is used to test the Unchecked option which we have in fluent */
+
+		[Test]
+		public void WhenRadioButtonFluentClick_01_Unchecked()
+		{
+			NavigateToSample("RadioButton", "Fluent");
+
+			TakeScreenshot("Before Tap");
+			var FluentUncheckRadioButton = new QueryEx(x => x.All().Marked("RadioButton_Fluent_Unchecked"));
```
Comment (Xiaoy312) [inline Uno.Gallery/Uno.Gallery.UITest/Given_RadioButton_02_Fluent.cs:23]:
```suggestion
			var fluentUncheckRadioButton = new QueryEx(x => x.All().Marked("RadioButton_Fluent_Unchecked"));
```

File: Uno.Gallery/Uno.Gallery.UITest/Given_RadioButton_02_Fluent.cs [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+			App.ScrollDownTo("RadioButton_Fluent_Unchecked");
+
+			FluentUncheckRadioButton.Tap();
+			TakeScreenshot("After Tap");
+			Assert.IsTrue(FluentUncheckRadioButton.GetDependencyPropertyValue<bool>("IsChecked"));
+
+		}
+		[Test]
+
+		public void WhenRadioButtonFluentClick_02_Checked()
```
Comment (Xiaoy312) [inline Uno.Gallery/Uno.Gallery.UITest/Given_RadioButton_02_Fluent.cs:34]:
```suggestion
			Assert.IsTrue(FluentUncheckRadioButton.GetDependencyPropertyValue<bool>("IsChecked"));
		}

		[Test]
		public void WhenRadioButtonFluentClick_02_Checked()
```

**Review by Xiaoy312 — COMMENTED** [review]
"test:" would be a more fitting category

File: Uno.Gallery/Uno.Gallery.UITest/TestBase.cs [hunk 3]
```diff
@@ -25,9 +26,11 @@ static TestBase()
             AppInitializer.TestEnvironment.AndroidAppName = Constants.AndroidAppName;
             AppInitializer.TestEnvironment.iOSDeviceNameOrId = Constants.iOSDeviceNameOrId;
             AppInitializer.TestEnvironment.CurrentPlatform = Constants.CurrentPlatform;
+			AppInitializer.TestEnvironment.ChromeDriverPath = @"C:\Program Files\Google\Chrome\chromedriver-win64";
```
Comment (Xiaoy312) [inline Uno.Gallery/Uno.Gallery.UITest/TestBase.cs:29]: move this to the `Constants`

**Review by Xiaoy312 — APPROVED** [review]

File: Uno.Gallery/Uno.Gallery.UITest/TestBase.cs (hunk 3 above)
Comment (kazo0) [reply Uno.Gallery/Uno.Gallery.UITest/TestBase.cs]: This should only be needed for your case I believe, we shouldn't include it as part of the source

Comment (Darsh0307) [reply Uno.Gallery/Uno.Gallery.UITest/TestBase.cs]: > This should only be needed for your case I believe, we shouldn't include it as part of the source
Should i remove the ChromeDriverPath ? @kazo0

**Pull request merged**